easybuild/easyconfigs/u/udocker/udocker-1.1.3-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'udocker'
+version = '1.1.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/indigo-dc/udocker'
+description = """A basic user tool to execute simple docker containers in batch
+ or interactive systems without root privileges."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/indigo-dc/udocker/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['5276f3d467f9533234239a022b094d37639b7090acd5e116304a488c57307995']
+
+dependencies = [('Python', '2.7.14')]
+
+# use_pip = True
```

Comment:
@SimonPinches Any specific reason for not using `pip` for the installation?

Reply:
@boegel No, specific reason for not using `pip` other than just assuming that EB would do the "right thing" by default!  (And it seems to work.)

Reply:
We are trying to move towards using `pip`, since it's the recommended Python installation tool.

There are some plans to make installation with `pip` the default in EasyBuild v4.0, see https://github.com/easybuilders/easybuild/issues/447 (but problems like https://github.com/easybuilders/easybuild-easyconfigs/issues/7491 have casted some doubt on that, to be honest...)

Reply:
@boegel I've pushed a new commit to use `pip` for this installation.